cmd/drone-docker-gcr/main.go

```diff
@@ -19,7 +19,7 @@ func main() {
 			"PLUGIN_JSON_KEY",
 			"GCR_JSON_KEY",
 			"GOOGLE_CREDENTIALS",
-			"GOOGLE_APPLICATION_CREDENTIALS",
+			"TOKEN",
```

Comment:
But we are always prefixing this kind of env variable names.

Reply:
Do you mean to prefix it with `PLUGIN_TOKEN`?

I was thinking about this from a `.drone.yml` standpoint.

0.4:
```yaml
publish:
  gcr:
    repo: foo/bar
    token: >
      $$TOKEN
```

0.7+:
```yaml
pipeline:
  gcr:
    image: plugins/gcr
    repo: foo/bar
    secrets: [token]
```

But I'm ok with removing `TOKEN` as well :)